README.md

```diff
@@ -8,7 +8,7 @@
 
 VitePress is [VuePress](https://vuepress.vuejs.org)' spiritual successor, built on top of [vite](https://github.com/vitejs/vite).
 
-Currently, it is in the `beta` stage. It is already suitable for out-of-the-box documentation use. There still might be issues, however, we do not plan to introduce any breaking changes from here on until the stable release. 
+Currently, it is in the `beta` stage. It is already suitable for out-of-the-box documentation use. We do not plan to introduce any breaking changes from here on until the stable release. 
```

Comment:
Shouldn't be in `release candidate`?

Reply:
oops, I've just fix at [2791e96](https://github.com/vuejs/vitepress/pull/2758/commits/2791e96345fa9521f4fba95fe2d0be7db3b3d58b)